src/app/admin/admin-reports/filtered-items/option-vo.model.ts

```diff
@@ -0,0 +1,46 @@
+import { Observable } from 'rxjs';
+
+export class OptionVO {
```

Comment:
Please add a comment to describe this class as well.  I'm not sure I understand what "OptionVO" means

Reply:
Done.